browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -35,6 +36,9 @@ namespace component_updater {
 
 namespace {
 
+const base::Feature kFeatureUpdateClientUseJSON{
+  "UpdateClientUseJSON", base::FEATURE_DISABLED_BY_DEFAULT};
```

Comment:
This was copied in from the normal updater configurator but it didn't end up being needed and was removed here:
https://github.com/brave/brave-core/pull/1117/commits/b961a8464d115712b3351043d2aa79d4418746c1